src/WheelsAndGoods.Api/Program.cs

```diff
@@ -1,3 +1,4 @@
+using FluentValidation;
```

Comment:
Why we add FluentValidator in program.cs?

Reply:
Good comment, will removed.